### PR TITLE
:responsiveness: using *media query*

### PR DESCRIPTION
**Pull request by aashimgarg**
Made the announcement page timeline responsive.

Worked on #495

### CODE REVIEW

**Comment by anushbhatia** [comment]
Screenshot_20200424_115504_com android chrome
See the g and c here they are getting overlapped

File: announcements.html [hunk 1]
```diff
@@ -79,7 +82,9 @@ <h5>June 20, 2020</h5>
 		  <div class="timeline-content">
 			<h3>Lorem ipsum dolor</h3>
 			<hr class="mw-100" style="margin-top:.5rem; margin-bottom:.75rem; background-color:#f47621"/>
-			<p>Lorem ipsum dolor sit amet, quo ei simul congue exerci, ad nec admodum perfecto mnesarchum, vim ea mazim fierent detracto. Ea quis iuvaret expetendis his, te elit voluptua dignissim per, habeo iusto primis ea eam.</p>
+
+			<p class="description">Lorem ipsum dolor sit amet, quo ei simul congue exerci, ad nec admodum perfecto mnesarchum, vim ea mazim fierent detracto. Ea quis iuvaret expetendis his, te elit voluptua dignissim per, habeo iusto primis ea eam.</p>
```
Comment (aashimgarg) [inline announcements.html:86]: why the class description is added to it? @anushbhatia

Comment (aashimgarg) [reply announcements.html:86]: i did not added to it.

Comment (anushbhatia) [reply announcements.html:86]: While resolving it might have occurred

**Pull request merged**